esp/applications/eclwatch/eclwatch.yaml

```diff
@@ -0,0 +1,175 @@
+eclwatch:
+   WsSMC:
+      ActivityInfoCacheSeconds: 10
+      ActivityInfoCacheAutoRebuildSeconds: 120
+   WsWorkunits:
+      ViewTimeout: 1000
+      LayoutProgram: dot/dot -Tsvg -Gordering=out
+      EclServer: ''
+      EclServerQueue: ''
+      StyleSheets:
+        xslt:
+        - name: atts
+          ^: "/esp/xslt/atts.xslt"
```

Comment:
I'm not familiar with this yaml syntax. What does it mean?

Reply:
This is at least temporarily there for backward compatability.  It creates a node like:  <xslt name="atts">/esp/xslt/atts.xslt</xslt>.  I'm trying not to change the content as part of this pull request, that can come later.

Reply:
Is this standard yaml? Or an extension in our jptree code?

Reply:
The YAML is valid although it's unlikely anyone else would use this syntax.  For PTREE it would result in an attribute name we couldn't support.  I use that special name to load it into the ptree in a unique way internally, but it gets written back out the same or similar to how you see it here.

So for YAML it's probably very unique, but legal.  I only use it in cases where I need backward compatability.  We can change this when we clean up what the services expect.  In fact this can probably be removed as it's from legacy eclwatch.. but we can focus on that in a follow on PR or point release.